### ISSUE TITLE
RTX 20xx Series Graphics Card Support

### ISSUE THREAD

**Issue by ThreeDlife7** [CloudCompare/CloudCompare#968]
**Describe the bug**

I can't seem to get this to run on my computer....
https://www.danielgm.net/cc/
I tried launching it with Windows 8 compatibility mode but no dice

Dean Manelis 2:13pm
It's blowing up on the QT library driver.
![image (1)](https://user-images.githubusercontent.com/56369562/66522487-c4e20f00-eaaa-11e9-98b9-aec8182d4afc.png)

Remove the app using add/remove programs. Make sure the folder is completely gone afterward. They right-click on the installer and select run as administrator.
Looks like this same library is used for a lot of game engines like Steam, seems a little problematic.
 
Jeremiah Fyler 2:25pm 
Oh ok, let me try that
Hmmmm still won't open

Dean Manelis 2:28pm
Yeah, same error.
Gimme a minute here...

Dean Manelis 2:37pm
Hmm, I found where someone else was having the same issue with an Nvidia rtx20xx series card. Can you try the "stereo" version?

Jeremiah Fyler 2:44pm 
Sure I can try that one
Damn, that version doesnt work either

Dean Manelis 2:57pm
The other person that had an issue never got an answer. My research shows it may have something to do with your video card and/or the 4k display.

**How to reproduce**
Try launching the software with a computer that has a RTX 20xx Series graphics card installed.

Steps to reproduce the behavior - include images if relevant:
-->

**Expected behaviour**

**Additional context**

**Your environment**
 - CC Version (or git hash/tag): CloudCompare_v2.10.2_setup_x64
 - OS & Version: Windows 10 
 - Graphics card: NVIDIA GeForce RTX 2070

 - Qt Version (if compiling):

**Comment by dgirardeau** [comment]
I wish I had one of these cards to test and reproduce the issue 😇 

I bet you don't have a secondary Intel chipset? (in which case you have to make sure the NVidia card is used, as Qt refuses to run on some old chipsets).

**Comment by ThreeDlife7** [comment]
No, the motherboard of my computer does not support an onboard chipset.

**Comment by dgirardeau** [comment]
Hum... Not sure what we can do...

Do you have some advanced options in the NVidia control panel that you can play with? Maybe some safety features / debug mode that can be enabled or disabled as with Quadro cards?

**Comment by jedfrechette** [comment]
I tested with the following setup and everything seems to work fine.

CC version: 2.10.2 (7z Archive)
OS: Windows 10 Pro
GPU: Nvidia Geforce 2080TI
GPU Drivers: 445.87
